src/status_im/contexts/shell/share/wallet/view.cljs

```diff
@@ -47,7 +48,7 @@
                   :button-label      (i18n/label :t/display)
                   :on-save           (fn [chain-ids]
                                        (rf/dispatch [:hide-bottom-sheet])
-                                       (reset! selected-networks (map #(get utils/id->network %)
+                                       (reset! selected-networks (map #(get network-utils/id->network %)
```

Comment:
:+1:

Reply:
aren't we replacing atoms with use-states?

Reply:
yes but this code already existed and refactoring it is a bigger job/ leaves room to introduce bugs